src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -114,10 +117,11 @@ bool Flag_Lightbulb( const int i, const int j, const int k, const int lv, const
 
 // TODO: fine-tune the criteria
 // (1) always refined to highest level in the region with r < 30 km
+   if (  ( r < amr->dh[lv] )  &&  ( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )  )
```

Comment:
Since in the postbounce phase, the inner region is always refined to highest level. I am wondering if we still need this. If yes, what is its effects on the refinement and performance. Could you check the performance between runs with and without this condition, and the sliceplot of the innermost region after 10-20 timestpes.

Reply:
I removed this part since it will always refine to the highest level as you said.